gp-ant-task/example/build.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="utf-8"?>
+<project name="gp-ant-task-1.1.5-SNAPSHOT" default="download-translations" 
+	xmlns:gp="antlib:com.ibm.g11n.pipeline.ant">
+	
+	<taskdef  uri="antlib:com.ibm.g11n.pipeline.ant" resource="com/ibm/g11n/pipeline/ant/antlib.xml">
+		<!-- If the gp-ant-task jar is not added to ANT_HOME/lib folder, 
+			then you may include the classpath for the jar like in the comment below-->
+		<!--<classpath path="lib/gp-ant-task-1.1.5-SNAPSHOT-with-dependencies.jar"/>-->
+	</taskdef>
+	
+	<!--Specify the location of the credentials json file-->
+	<property name="gp.credentials" value="src/credentials.json"/>
+	
+	<!-- basic usecase for uploading resources(this uploads java .properties files by default)-->
+    <target name="upload-resources">
+        <gp:upload credentialsJson= "${gp.credentials}" sourceDir="src/main/resources">
+        </gp:upload>
+    </target>
+	
+	<target name="upload-resources-adv">
+	        <gp:upload credentialsJson= "${gp.credentials}" sourceDir="src/main/resources">
+	        	<targetLanguage lang="fr"/>
+	        	<targetLanguage lang="ko"/>
+	        	<targetLanguage lang="pt-BR"/>
+	        	<targetLanguage lang="es"/>
+	        	<bundleSet  type="JSON">
+	        		<fileset dir="src/main/resources" includes="**/*.json"/>
+	        	</bundleSet>
+	        </gp:upload>
+	    </target>
+    
+    <!-- basic use case for downloading resources (this downloads the java .properties files to the target/classes folder of the project) -->
+    <target name="download-translations">
+        <gp:download credentialsJson="${gp.credentials}" sourceDir="src/main/resources" outputDir="target/classes">
+        </gp:download>
+    </target>
+        	
+    <!-- advanced use case for downloading resources. This downloads java properties files and json properties files and organizes them differently -->
+    <target name="download-translations-adv">
+    	<property name="url" value="https://gp-rest.ng.bluemix.net/translate/rest"/>
```

Comment:
I'd recommend to use prefix "gp" for all of these properties.
gp.url, gp.userId, gp.password, gp.instanceId...